### ISSUE TITLE
It would be great to add an RSS feed.

### ISSUE THREAD

**Issue by juliend2** [slowernews/slowernews#4]

**Comment by slowernews** [comment]
As an heavy user of RSS I agree with you. I'll make some research on how to make one.

The only thing that bugs me is that most articles are not listed in a random order and some are even related with the previous/next one. When you receive a feed you loose that.

**Comment by juliend2** [comment]
@slowernews Yes, I think I know what you mean.

Then maybe it would be better to have several xml feeds (one for each category), to mitigate that?

Let me know if you need help with the implementation.

**Comment by klimcio** [comment]
Or maybe a newsletter? Given the fact that I learned about it from a Hackernews Newsletter.

**Comment by slowernews** [comment]
Hi @klimcio I'm not refusing the idea yet but I see some points against a newsletter:

- It forces me on a schedule (monthly, quarterly, etc) and a structure. My intention is to add articles only when relevant and in whatever cathegory they happen to be.

- It adds some stress on some readers (I'm in favour of [Inbox Zero](https://github.com/slowernews/hamster-system)). I prefer that people read Slowernews whenever they feel like.

- I don't just add articles. Sometimes I deprecate/delete/substitute some when they are not relevant anymore or when I find a better one. I can't do this in a newsletter.

**Comment by klimcio** [comment]
Hi @slowernews 

I agree with Inbox Zero and reading whenever people feel like.

I was thinking about a newsletter that's not scheduled but sent whenever it is relevant. 

All the changes you mentioned might be in a mail like that as well: Article X no longer relevant got deleted, Article Y substitutes for article A because it is more informative.

Although, it's just a git log then, only sent by mail.

**Comment by RobKohr** [comment]
I came here to comment that this needs an RSS feed. I tried to add it to my reader, and my reader said nope. 

I am unlikely to remember to come back in a month, so something, and email, and feed, anything would help

**Comment by slowernews** [comment]
Hello, I just created a rss feed. Some notes: 

- After trying some apps, I realized that it would be simpler to make it manually based on a existent feed. New articles will be scarce so updating it manually is manageable.

- There will be only one feed for each page (for now at least). Didn't seem practical to have 28 categories that will be updated rarely. The article category will be mentioned in the title.

- I only tested on Feedly and it's working. As I understand Feedly takes some time to update a feed so for now you'll see an old feed. Might be better to try tomorrow instead.

- My knowledge of feeds is residual so feel free to suggest changes if you notice rookie mistakes.
